src/plugins/commander.go

```diff
@@ -96,16 +96,7 @@ func (c *Commander) agentBackoff(agentConfig *proto.AgentConfig) {
 	}
 
 	backOffSettings := sdk.ConvertBackOffSettings(agentConfig.GetDetails().GetServer().GetBackoff())
-
 	c.cmdr.WithBackoffSettings(backOffSettings)
-	err := c.cmdr.Close()
```

Comment:
Hey, why was the close removed from here and metrics_sender ?

Reply:
Some of these changes are from my PR here https://github.com/nginx/agent/pull/401
The reconnect of grpc clients after the backoff settings are updated isn't actually necessary so we can stop doing this.

Reply:
Aww right, that makes sense got confused as to what change in this PR caused it to not be needed. Thank you!